src/Suave/Http.fsi

```diff
@@ -91,9 +91,12 @@ module Http =
   /// Functions have signature f :: params... -> HttpContext -> HttpContext.
   module Writers =
 
-    /// Sets a header with the key and value specified
+    /// Adds or replace a headers with the key and value specified
     val setHeader : key:string -> value:string -> WebPart
```

Comment:
I'm not onboard with the doc change, as it seems more accurate before. How about "Ensures that the `key` header has the given `value`", instead? And for `addHeader` (renamed from `putHeader`): "Adds the header `key` with the given `value` to the list of returned headers, even if that header already exists"